eng/pipelines/templates/jobs/new.smoke.tests.yml

```diff
@@ -0,0 +1,32 @@
+parameters:
+  - name: ServiceDirectory
+    type: string
+    default: ''
+  - name: Daily
+    type: boolean
+    default: true
+  - name: TimeoutInMinutes
+    type: number
+    default: 60
+
+jobs:
+  - job: SmokeTest
+    steps:
+      - ${{ if eq(parameters.Daily, true) }}:
```

Comment:
Is it worth separating these vs. calling a single script with a parameter `-Daily`?

Reply:
I think putting the daily parameter into the script that runs the smoke tests is worthwhile, I can make that change

Reply:
I have it set up so passing in `-serviceDirectory .` will run the Daily version, and passing in a serviceDirectory (ie `data/aztables`) will run smoke tests only for that package. Thoughts on this approach?

Reply:
I think eventually we'll want to have the option to run the GA/beta validation against ALL packages rather than just a single service directory, so having the explicit boolean makes it a little clearer what the behavior is going to be and easier to update in the future.